docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSample.cs

```diff
@@ -52,12 +58,14 @@ public static void Example()
 
                 if (result.PredictedLabel)
                     // The i-th sample is predicted as an inlier.
-                    Console.WriteLine("The {0}-th example with features [{1}] is an inlier with a score of being inlier {2}",
-                        i, featuresInText, result.Score);
+                    Console.WriteLine("The {0}-th example with features [{1}]" +
+                        "is an inlier with a score of being inlier {2}", i,
+                            featuresInText, result.Score);
```

Comment:
```suggestion
                        featuresInText, result.Score);
```
Would you think this change makes things better?

Reply:
I'm not sure. I was looking at the Clustering formatting changes @artidoro made and I think he indented after every break, other than comments, but I'll find out and change it.